HHS/OPRE-OPS
# fix: Agreement Summary Cards in edit mode

## What changed

Fixes Agreement Summary Cards in Agreement BLIs Tab (in edit mode)

## Issue

#2193 #2325 

## How to test

1. Goto Agreement 1
2. Goto SC+BLI tab
3. Enter edit mode
4. toggle show DRAFT
5. see amounts change
6. add some more DRAFT BLIs
7. see amounts change

## Screenshots

https://github.com/HHS/OPRE-OPS/assets/4629398/1afab7ae-f34e-4422-9403-9dcc3811ff73
